x-pack/legacy/plugins/canvas/external_runtime/types.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore Unlinked Webpack Type
+import ContainerStyle from 'types/interpreter';
+
+export interface CanvasWorkpad {
```

Comment:
Should this just bring in the existing types from `/canvas/types`?

Reply:
Will address.